src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -571,6 +583,10 @@ private async Task ValidateHostRuntime(bool isInProc, bool isVerbose)
             throw new CliException($"Invalid host runtime '{HostRuntime}'. Valid values are 'default', '{DotnetConstants.InProc8HostRuntime}', '{DotnetConstants.InProc6HostRuntime}'.");
         }
 
+        private void ThrowIfInProc()
```

Comment:
Using a "Throw**If**" naming is typically when the function itself will perform the conditional check to throw/not throw. Considering that is how it is used, maybe you could move the `if (Utilities.IsMinifiedVersion())` check to within this function?

Reply:
How about changing the name to something like `ThrowForInProc()` or `ThrowForInProcCase()`. To provide more context, `minified version` means the coretools that goes with Visual studio and it is slightly different than the cli version (it is minified by removing workers to trim its size). I feel in future, other parts of code can also use this check, hence keeping it in utilities seems right.

Reply:
`ThrowForInProc()` works for me.